packages/seal-redisearch-adapter/src/RediSearchSchemaManager.php

```diff
@@ -31,7 +31,7 @@ public function existIndex(Index $index): bool
         try {
             $indexInfo = $this->client->rawCommand('FT.INFO', $index->name);
         } catch (\RedisException $e) {
-            if ('Unknown Index name' !== $e->getMessage()) {
+            if ('unknown index name' !== \strtolower($e->getMessage())) {
```

Comment:
This seems now be returned differently based on the redis server version so we need to strtolower now.